python/lsst/sims/maf/slicers/baseSpatialSlicer.py

```diff
@@ -98,7 +98,7 @@ def _sliceSimData(islice):
                 indices = self.opsimtree.query_ball_point((sx, sy, sz), self.rad)
 
             for key in self.slicePoints.keys():
-                if np.size(self.slicePoints[key]) > 1:
+                if (np.size(self.slicePoints[key]) > 1) & (key is not 'bins') & (key is not 'binCol'):
```

Comment:
yeah, this is abusing the slicePoint a bit, but it's such an easy way to pass extra info to the metric.  I made it a little more elegant so that the "don't slice me" keys are all listed.